### ISSUE TITLE
Alice Authentication Conflict with Ant Mobi 2.0 app

### ISSUE THREAD

**Issue by vinitshetty** [krishnavelu/alice_blue#425]
Current API authentication works fine and we can use Ant App and website in parallel to algo.

But when I use Ant Mobi 2 app, algo token becomes invalid and authentication failed on the algo side. 

This means we cannot use Ant Mobi 2 app and algo together.

I have raised this with Alice and they have asked us to change to their new APIs and new authentication mechanism.

It will be great if we could handle this. 

https://v2api.aliceblueonline.com/introduction

**Comment by jaydave1988** [comment]
Duplicate of #418
